examples/jsm/renderers/webgpu/WebGPUNodeBuilder.js

```diff
@@ -0,0 +1,89 @@
+import { NodeBuilder } from '../../nodes/core/NodeBuilder.js';
```

Comment:
I guess it would be good to avoid all dependencies to `examples/jsm/nodes/`. The idea was start with a fresh system (and only with the minimal bits) which are required for a basic material.

It seems there are dependencies to WebGL related logic in `NodeBuilder` which should not be part of the new system. I also suppose that certain parts of the node material system will look different that what is existing now in `examples/jsm/nodes/` when devs contribute to it. Hence, it seems better starting from scratch in `examples/jsm/renderers/webgpu/node/` with the minimal setup.

Reply:
> The idea was start with a fresh system (and only with the minimal bits) which are required for a basic material.

Yes, in fact, just this still not ready.. only however is that we have an opening that I would like take advantage... I think in creating an `examples/jsm/renderers/nodes` new `NodeMaterial` system for hybrid renderers and a NodeBuilder for each Renderer, for example... `..renderers/webgl/WebGLNodeBuilder` and `..renderers/webgpu/WebGPUNodeBuilder` extended of `renderers/nodes/core/NodeBuilder`. Any single `Node` can be used in different renderers and WebGL versions. Part of the work I will do the disassociation of WebGL and cleanup.

Reply:
I will try to be as minimal as possible...

Reply:
It's a meaningful goal to have a rendering agnostic description of materials. So the a node composition itself should have no WebGL or WebGPU associated dependencies. Instead, specific builders produce the respective WebGL or WebGPU output.

Reply:
I think this the idea but that is no ready I ask you to ignore this version of NodeBuilder for the time being.

Reply:
A new version of NodeBuilder going to be moved to `examples/jsm/renderers/nodes` for have no WebGL or WebGPU dependencies as the comments above... until the release of this PR.

>  I think in creating an examples/jsm/renderers/nodes new NodeMaterial system for hybrid renderers and a NodeBuilder for each Renderer, for example... ..renderers/webgl/WebGLNodeBuilder and ..renderers/webgpu/WebGPUNodeBuilder extended of renderers/nodes/core/NodeBuilder